src/main/java/games/strategy/engine/lobby/server/db/HashedPassword.java

```diff
@@ -15,6 +19,14 @@ public HashedPassword(final String hashedPassword) {
    * Returns true if the hashed password looks like it could be a hash.
    */
   public boolean isValidSyntax() {
+    return isMd5Crypted() || isBcrypted();
+  }
+
+  public boolean isBcrypted() {
```

Comment:
This method needs to be accessed from the `LobbyLoginValidator` class to determine wether or not the given string should be BCrypted...